blt/src/Blt/Plugin/Commands/HsCommands.php

```diff
@@ -50,11 +53,41 @@ public function postCodeDeployUpdate() {
         $failed[] = $site;
       }
     }
+    unlink(sys_get_temp_dir() . '/update-report.txt');
+
     $this->yell(sprintf('Updated %s sites successfully.', count($success)), 100);
+
     if ($failed) {
       $this->yell(sprintf("Update failed for the following sites:\n%s", implode("\n", $failed)), 100, 'red');
+
+      if (EnvironmentDetector::isAhStageEnv() || EnvironmentDetector::isAhProdEnv()) {
```

Comment:
Is there a reason you're using these instead of `$target_env`?

Reply:
Not at all. I didn't really think about it. lol